netbout-web/src/main/java/com/netbout/dynamo/DyAlias.java

```diff
@@ -70,15 +81,6 @@
      */
     private final transient Item item;
 
-    static {
```

Comment:
@mkordas  It is with checkstyle, because it wants me to refer to the variable as DyAlias.MAIL, but if I do that it is a syntax error in java saying that I cannot assign a value to a final filed, so I did the initialization directly no need for   ```static{}```